test/src/specs/image/image_widget_test.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mix/mix.dart';
+
+void main() {
+  group('StyledImage', () {
+    TestWidgetsFlutterBinding.ensureInitialized();
+
+    testWidgets('receive all the attributes', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: StyledImage(
+              style: Style(
+                image.width(152),
+                image.height(152),
+                image.color.black(),
+                image.repeat.repeat(),
```

Comment:
These cases are always a bit tough, but they seem redundant, can we have the call method on `image.repeat()` be ImageRepeat.repeat?